### ISSUE TITLE
Follow user's following instead of followers

### ISSUE THREAD

**Issue by mifi** [mifi/SimpleInstaBot#57]
Feature request received by email:
> If I focus in jewelry companies on IG, account following contains jewelry IG account, followers are usually just random people. SO I want to follow same IG account as a specific jewelry account follows.

**Comment by mikikuke** [comment]
yes yes, that is my email, please do this quick, people we need to support this guy

**Comment by garrettkidd1** [comment]
Also would love this. very important as when its doing following, they are business accounts but people. So please have it as an option to use followers instead. Keep me updated
